doc/api/debugger.md

```diff
@@ -194,11 +194,19 @@ debug>
   after)
 * `watch(expr)`: Add expression to watch list
 * `unwatch(expr)`: Remove expression from watch list
+* `unwatch(index)`: Remove expression at specific index from watch list
```

Comment:
Is this addition intentional? It doesn't appear in the internal help. And an index is an expression, no?

Reply:
@Trott Sorry for the confusion. It is from https://github.com/nodejs/node/pull/42882 and not related to this PR. I removed.